templates/navigation-bar-new.twig

```diff
@@ -42,46 +42,27 @@
 					</a>
 
 					{% if item.children is not empty %}
-						<nav class='nav-submenu'>
-							<div class='nav-submenu-wrapper'>
-								<ul>
-								{% for key,item in item.children %}
-									<li>
-										{% if fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == act_page_id %}
-											{% set link_ga_action = 'Act' %}
-										{% elseif fn('get_post_meta', item.ID, '_menu_item_object_id', true ) == explore_page_id %}
-											{% set link_ga_action = 'Explore' %}
-										{% else %}
-											{% set link_ga_action = item.title %}
-										{% endif %}
-										<a
-											class="nav-link"
-											href="{{ item.get_link }}"
-											data-ga-category="Submenu Navigation"
-											data-ga-action="{{ link_ga_action }}"
-											data-ga-label="{{ page_category }}">
-												{{ item.title }}
-										</a>
-									</li>
-								{% endfor %}
-								</ul>
-							</div>
-						</nav>
+						{% include 'navigation-submenu.twig' with { menu: item } %}
 					{% endif %}
 				</li>
 				{% endfor %}
-				<li class="nav-item nav-donate">
+
+				{% set donate_menu = donate_menu_items[0] ?? {} %}
+				<li class="nav-donate">
 					<a
-						class="btn btn-donate"
-						href="{{ donatelink }}"
+						class="btn btn-donate {{ donate_menu.children ? 'dropdown' : '' }}"
+						href="{{ donate_menu.link }}"
 						data-ga-category="Menu Navigation"
 						data-ga-action="Donate"
 						data-ga-label="{{ page_category }}">
-							{{ donatetext }}
+							{{ donate_menu.title }}
 					</a>
-				</li>
+					{% if donate_menu.children is not empty %}
+						{% include 'navigation-submenu.twig' with { menu: donate_menu } only %}
```

Comment:
Why do you use `only` here? 🤔 We don't use it anywhere else, including in the other [inclusion of this same template](https://github.com/greenpeace/planet4-master-theme/pull/1626/files#diff-3ad24a02d9e63bde53c8195a030484bfb200dac551677c03584386ba3394c92aR45)